### PR TITLE
Added a few more gamemodes

### PR DESCRIPTION
**Pull request by Arrow768**
Adds a few more game modes as requested in https://forums.aurorastation.org/viewtopic.php?f=21&t=8798&p=82273#p82273

### CODE REVIEW

File: code/game/gamemodes/mixed/feeding.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/datum/game_mode/feeding
+	name = "Feeding (Vamp + Ling)"
+	round_description = "The the crew is being fed upon!"
```
Comment (Lohikar) [inline code/game/gamemodes/mixed/feeding.dm]: The the.

File: code/game/gamemodes/mixed/paranormal.dm [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/datum/game_mode/paranormal
+	name = "Paranormal (Cult + Vamp)"
+	round_description = "Paranormal activities occur on the station."
+	extended_round_description = "Vampires and a Clut spawn in this mode."
```
Comment (Lohikar) [inline code/game/gamemodes/mixed/paranormal.dm]: Clut.

File: code/game/gamemodes/mixed/paranormal.dm [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/datum/game_mode/paranormal
+	name = "Paranormal (Cult + Vamp)"
```
Comment (Lohikar) [inline code/game/gamemodes/mixed/paranormal.dm]: This style is inconsistent with the other gamemodes - there should be no spacing around the `+`.

File: code/game/gamemodes/mixed/infiltration.dm [hunk 4]
```diff
@@ -0,0 +1,11 @@
+/datum/game_mode/infiltration
+	name = "Infiltration (Malf + Ninja)"
```
Comment (Lohikar) [inline code/game/gamemodes/mixed/infiltration.dm]: Same here.

File: code/game/gamemodes/mixed/feeding.dm [hunk 5]
```diff
@@ -0,0 +1,11 @@
+/datum/game_mode/feeding
+	name = "Feeding (Vamp + Ling)"
```
Comment (Lohikar) [inline code/game/gamemodes/mixed/feeding.dm]: And here.

**Comment by Lohikar** [comment]
...do we also really need more mixed modes?

**Comment by Arrow768** [comment]
Well, we dont have a single one with vampires in them, so I would argue yes.

**Comment by Lohikar** [comment]
`Paranormal`'s name is a little too close to `Paranoia` imo too.

**Comment by Arrow768** [comment]
Suggestions for a better name then ?

**Comment by skull132** [comment]
@Arrow768 `Veil Party` or something relating to the `Veil`. Both Vampire and Cult are from the Veil.

**Comment by skull132** [comment]
\> Veil party is now an official gamemode
👌 

Lessgo!

**Pull request merged**